detector/detector.go

```diff
@@ -157,62 +157,69 @@ func (d *Detector) Run(ctx context.Context) ([]output.Dashboard, error) {
 		d.datasourcePluginIDs[ds.Name] = ds.Type
 	}
 
-	dashboards, err := d.grafanaClient.GetDashboards(ctx, 1)
-	if err != nil {
-		return []output.Dashboard{}, fmt.Errorf("get dashboards: %w", err)
-	}
+	var page int
+	for {
+		page += 1
+		dashboards, err := d.grafanaClient.GetDashboards(ctx, page)
+		if err != nil {
+			return []output.Dashboard{}, fmt.Errorf("get dashboards: %w", err)
+		}
+		if len(dashboards) == 0 {
```

Comment:
So whenever there is a dashboard it will keep running, right? PageSize is 5K?

Reply:
Yes. Page size is 5000 (see [here](https://github.com/grafana/detect-angular-dashboards/blob/bc65fdbac7cf9c65c191b5617c07d5f53d91f081/api/grafana/grafana.go#L44)), and it loops until there are results returned, incrementing the page each time.